### ISSUE TITLE
Could not find fr.acinq.lightning:lightning-kmp:1.6.2-FEECREDIT-4.

### ISSUE THREAD

**Issue by rushmi0** [ACINQ/phoenixd#10]
I don't understand why it doesn't work?

![image](https://github.com/ACINQ/phoenixd/assets/120770468/e0526057-ca65-47fc-8492-ffb4b5832e81)

**Comment by vincenzopalazzo** [comment]
phoenix and phoenixd is using lightning-kmp that it is a lightning implementation.

So long story short to fix the problem you need just 

```
git clone https://github.com/ACINQ/lightning-kmp.git && cd lightning-kmp
git checkout v1.6.2-FEECREDIT-4
./gradlew :publishToMavenLocal
```

and then the error will be fixed 😄

**Comment by dpad85** [comment]
This is a temporary issue. This version should be available on the maven central repository (so you won't have to build it yourself), but there's a problem with tests on iOS platforms that we need to investigate and fix first before we can do that.

**Comment by pm47** [comment]
Fixed by #18.
